tests/test_flyer_with_panda.py

```diff
@@ -0,0 +1,221 @@
+        writers[1],
+        name="detector_2",
+        writer_timeout=3,
+    )
+    return (detector_1, detector_2)
+
+
+@pytest.fixture
+async def panda():
+    async with DeviceCollector(sim=True):
+        sim_panda = PandA("PANDAQSRV:", "sim_panda")
+
+    assert sim_panda.name == "sim_panda"
+    yield sim_panda
+
+
+async def test_hardware_triggered_flyable_with_static_seq_table_logic(
+    RE: RunEngine,
+    detector_list: tuple[StandardDetector],
+    panda,
+):
+    """Run a dummy scan using a flyer with a prepare plan stub.
+
+    This runs a dummy plan with two detectors and a flyer that uses
+    StaticSeqTableTriggerLogic. The flyer and detectors are prepared with the
+    prepare_static_seq_table_flyer_and_detectors_with_same_trigger plan stub.
+    This stub creates trigger_info and a sequence table from given parameters
+    and prepares the fly and both detectors with the same trigger info.
+
+    """
+    names = []
+    docs = []
+
+    def append_and_print(name, doc):
+        names.append(name)
+        docs.append(doc)
+
+    RE.subscribe(append_and_print)
+
+    shutter_time = 0.004
+    exposure = 1
+    deadtime = max(det.controller.get_deadtime(1) for det in detector_list)
+
+    trigger_logic = StaticSeqTableTriggerLogic(panda.seq[1])
+    flyer = HardwareTriggeredFlyable(trigger_logic, [], name="flyer")
+
+    def flying_plan():
+        yield from bps.stage_all(*detector_list, flyer)
+
+        yield from prepare_static_seq_table_flyer_and_detectors_with_same_trigger(
+            flyer,
+            detector_list,
+            num=1,
+            width=exposure,
+            deadtime=deadtime,
+            shutter_time=shutter_time,
+        )
+
+        for detector in detector_list:
+            detector.controller.disarm.assert_called_once  # type: ignore
+
+        yield from bps.open_run()
+        yield from bps.declare_stream(*detector_list, name="main_stream", collect=True)
+
+        set_sim_value(flyer.trigger_logic.seq.active, 1)
+
+        yield from bps.kickoff(flyer, wait=True)
+        for detector in detector_list:
+            yield from bps.kickoff(detector)
+
+        yield from bps.complete(flyer, wait=False, group="complete")
+        for detector in detector_list:
+            yield from bps.complete(detector, wait=False, group="complete")
+
+        # Manually incremenet the index as if a frame was taken
+        for detector in detector_list:
+            detector.writer.index += 1
+
+        set_sim_value(flyer.trigger_logic.seq.active, 0)
+
+        done = False
+        while not done:
+            try:
+                yield from bps.wait(group="complete", timeout=0.5)
+            except TimeoutError:
+                pass
+            else:
+                done = True
+            yield from bps.collect(
+                *detector_list,
+                return_payload=False,
+                name="main_stream",
+            )
+            yield from bps.sleep(0.01)
```

Comment:
What is this sleep needed for?

Reply:
I think it is lingering from an old issue that has since been sorted. I just copied this plan format from the old stuff Rose did.

Reply:
I've removed it and the one in the flyer.